public/index.html

```diff
@@ -0,0 +1,11 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Open Table </title>
+  </head>
+  <body>
+    <div id="app"></div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
```

Comment:
If importing jquery via webpack, would suggest removing reference to CDN since you are downloading two redundant files and will slow performance